charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml

```diff
@@ -1,55 +0,0 @@
-{{- if not .Values.gep19Monitoring }}
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-controller-manager-observability-config
```

Comment:
Not introduced with this PR but we could remove https://github.com/gardener/gardener-extension-provider-gcp/blob/3f8278cebb69ead37b712a6ac8f8dda644c39615/pkg/controller/controlplane/valuesprovider.go#L324-L327
The corresponding ConfigMap was deleted in https://github.com/gardener/gardener-extension-provider-gcp/pull/754.